content/en/docs/instrumentation/net/getting-started/_index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Getting Started
```

Comment:
This page removes the console application example - there are definitely scenarios in which a console app (or service-like thing that happens to be a console app) are where people start. It also demonstrates how to acquire a tracer, which is done magically behind the scenes in the ASP.NET Core scenario.

Was that intentional? I'm not super opposed, but I think the two scenarios are very common for getting started and worth demonstrating.

Reply:
I have moved the console application example to the `getting-started/traces.md` file.
It was intentional but I'm debating whether I should just keep the tracing example on this index page (not create a new `traces.md` file), and then have the metrics and logs pages as they are in this PR.

I might need to change the title to Getting started with Tracing or at least call this out in the header of the page, that this example would just be for tracing in either .NET or .NET Framework and that the metrics and logging examples are location on these other pages.

Or would having the 3 pages separate and just leaving a not saying they exist (like what this PR is doing) would be better..

Reply:
Deciding on keeping the three pages for logs, traces, and metrics as separate and include relative links to them on this main getting started page. Let me know if you have any concerns with this approach.